code/workspaces/web-app/src/components/stacks/StackCardActions.js

```diff
@@ -95,6 +96,12 @@ export const PureStackCardActions = ({ stack, openStack, deleteStack, editStack,
             Edit
           </MenuItem>
         </PermissionWrapper>}
+        {(stack.type === 'rstudio' || stack.type === 'jupyterlab' || stack.type === 'jupyter' || stack.type === 'nbviewer' || stack.type === 'rshiny'
+          || stack.type === 'zeppelin') && deleteStack && ownsStack && <PermissionWrapper userPermissions={userPermissions} permission={deletePermission}>
+          <MenuItem disabled={shared} onClick={() => shareStack(stack, 'project')}>
```

Comment:
Here is where the value is hard coded - to change the sharing value to 'project'.